pkg/private/zip/build_zip.py

```diff
@@ -153,6 +155,7 @@ def add_manifest_entry(self, entry):
       entry_info.compress_type = zipfile.ZIP_STORED
       # Set directory bits
       entry_info.external_attr |= (UNIX_SYMLINK_BIT << 16)
+      src = src.encode('utf-8')
```

Comment:
Is this needed?  This seems to be done implicitly as of python 3.6 (https://docs.python.org/3.6/library/zipfile.html#zipfile.ZipFile.writestr).

I think we need to document somewhere what our minimum supported version of python is.  CentOS 7 and Ubuntu 18.04 LTS both have 3.6, so perhaps we should standardize on that?

Reply:
Added 3.6 as a requirement to developers.md.